### ISSUE TITLE
[Screen Reader- API Management -Change background]: In the process of selecting the background color, screen reader is not announcing the updated value for the current focused color.

### ISSUE THREAD

**Issue by JMach1** [paperbits/paperbits-demo#213]
### TL;DR

1. styles page
1. open a color editor
1. screen reader does not work properly with the color picker

### Original MS task
https://msazure.visualstudio.com/One/_workitems/edit/15430834

User Experience:
Users dependent on screen-reader to navigate the page will face difficulty in navigating the page if the modified information does not narrate by screen reader, hence user will face difficulty in navigating the page efficiently

Note: User credentials should NOT be included in the bug.
Repro-Steps:
Open Azure portal with valid credentials and create an API Management.
Select Developer Portal (Preview) button and select it.
Navigate to the URL:[API Management (azure-api.net)](https://apitestaug.developer.azure-api.net/new-page-copy)
Navigate to the Left nav and select Styles button.
Navigate to style page and select Change background button.
Navigate to the various controls of the "Change background" dialog box
Verify the issue
Actual Result:
In the process of selecting the background color, screen reader is not announcing the updated value for the current focused color.
Observation: Screen reader keep silent when user modified the value of background color.
Expected Result:
While modify the value of background color screen reader should be announcing the updated value.
EX: Screen reader should announce when user modified the value of background color.

Note: Same issue repro with all the screen readers.
MAS Reference:
[MAS 1.3.1 – Info and Relationships](https://aka.ms/MAS1.3.1)

Reference Links:
[Accessibility ](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-poyad%40microsoft.com%7cc11478591b5a474f6d9d08d7199726c0%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637006009300301864&sdata=ZTWYamT1yE7yko5QU35ZJLf/5ZydtDeGAKrQGUiugJs%3D&reserved=0)Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier. 
External Bug Process: If this bug belongs to external team, mark it resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDUwYzk1OWQ5MmNjZDRmZDA5MzBkYzA5OGQ5YzYzNmZhfDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3MjkxNTE1NDY2MjEyMTl8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=VXJwZXd5MnZRcEc0L0lIV1dBV0gvNThzNEt2MmphN2hlR2VOQ09JYzd1MD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-cthakur%40microsoft.com)sharepoint.com) “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”. 
Please reach out to [C&AI Teams ](https://nam06.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cv-dhti%40microsoft.com%7cf90880a914be401871f108d763233116%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637086876782361405&sdata=/ULS%2BbeqMjysCoiWHvFgAbSnuFJqxO3wtpHmlidPjbU%3D&reserved=0)channel for any process related queries.
